BrainPortal/cbrain_plugins/cbrain-plugins-base/cbrain_task/start_vm/bourreau/start_vm.rb

```diff
@@ -0,0 +1,245 @@
+
```

Comment:
General comment about this whole file: what does it produce after rdoc is run? Many methods are nodoc and some are not, is the selection of methods to document and methods to hide well established?